spec/mailers/previews/digest_mailer_preview.rb

```diff
@@ -1,6 +1,16 @@
 # Preview all emails at http://localhost:3000/rails/mailers/digest_mailer
 class DigestMailerPreview < ActionMailer::Preview
   def digest_email
-    DigestMailer.with(user: User.last, articles: Article.all).digest_email
+    user = User.last
+    tags = user.cached_followed_tag_names&.first(12)
+    first_billboard = Billboard.for_display(area: "digest_first",
+                                            user_id: user.id,
+                                            user_tags: tags,
+                                            user_signed_in: true)
+    second_billboard = Billboard.for_display(area: "digest_second",
+                                             user_id: user.id,
+                                             user_tags: tags,
+                                             user_signed_in: true)
+    DigestMailer.with(user: user, articles: Article.all, billboards: [first_billboard, second_billboard]).digest_email
```

Comment:
Is there a possibility that an identical billboard would be shown twice? Will that be a problem?